scripts/set_nixpath.sh

```diff
@@ -0,0 +1,6 @@
+update_NIX_PATH() {
+  local scriptDir=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+  export NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/$(nix-shell -p jq --run "jq .rev < ${scriptDir}/../nixpkgs-src.json -r").tar.gz
+  export NIX_PATH_LOCKED=1
```

Comment:
at one point, i was going to have the nix expressions detect that `NIX_PATH` was already configured, and not have them re-import the nixpkgs specified in `nixpkgs-src.json`

that change has since been removed, but i am keeping the variable for if i bring it back later